### PR TITLE
ci: Add isort

### PR DESCRIPTION
**Pull request by chivay**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -34,6 +39,11 @@ steps:
   image: pyfound/black:21.7b0
   commands:
     - black --check --diff drakrun/
+- name: isort check
+  image: python:3.8-buster
+  commands:
+    - pip3 install isort==5.9.3
+    - isort --profile black drakcore/
```
Comment (manorit2001) [inline .drone.yml:46]:
```suggestion
    - isort --profile black drakrun/
```

Comment (chivay) [reply .drone.yml]: thanks :d

**Comment by manorit2001** [comment]
Also, does `isort` return a non-zero exit code normally? As I could see a flag named `--check` in `isort` for doing this

**Comment by chivay** [comment]
> Also, does `isort` return a non-zero exit code normally? As I could see a flag named `--check` in `isort` for doing this

:+1:

**Review by kscieslinski — APPROVED** [review]

**Pull request merged**